GAE/src/org/waterforpeople/mapping/app/web/ProcessorServlet.java

```diff
@@ -162,4 +175,18 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
         }
 
     }
+    
+    private Long parseFormID(HttpServletRequest req) {
+        String formID = StringUtils.trim(req.getParameter("formID"));
+        if (StringUtils.isNotBlank(formID)) {
```

Comment:
Here you use this style difference `formID` vs `formId` because of some legacy reason in the app code?